### DISCUSSION TITLE
`ubuntu-latest` github builds broken because of `case-paths` dependency failing

### DISCUSSION THREAD

**Discussion by pacu** [pointfreeco/swift-parsing#336] [Q&A] [answered]
I'm building a library with `swift-parsing`. My Github actions that builds `ubuntu-latest` fails because of 
the `swift-case-path` failing to build.

link: https://github.com/pacu/zcash-swift-payment-uri/actions/runs/7311162836/job/19920429063?pr=31
````
/home/runner/work/zcash-swift-payment-uri/zcash-swift-payment-uri/.build/checkouts/swift-case-paths/Sources/CasePathsMacros/CasePathableMacro.swift:27: error: undefined reference to '$s11SwiftSyntax08EnumDeclB0VN'
````

**Comment by stephencelis** [comment]
Are you on Swift 5.9.2? 5.9.1 and earlier have an SPM bug where tests fail to build when depending on SwiftSyntax.

**Comment by pacu** [comment]
I'm specifying 5.9 in my `swift-setup` action, I'm going _to try_ get it to 5.9.2 and see how it goes. 

update: it appears that the swift-setup action does not have 5.9.2 yet 😰

**Comment by stephencelis** [comment, marked as answer]
If you want, you can lock your swift-case-paths to 1.0.0 for the time being while you wait. Since I don't think this is a bug with the library, though, I'm going to convert to a discussion. Good luck!

**Reply by pacu** [reply]
This solved the issue. For readers having the same problem, take this as a workaround. Proper fix is actually using the proper version of Swift.
